raiden-ts/tests/unit/epics/path.spec.ts

```diff
@@ -0,0 +1,334 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import { of, BehaviorSubject } from 'rxjs';
+import { bigNumberify } from 'ethers/utils';
+
+import { UInt, Address } from 'raiden-ts/utils/types';
+import {
+  newBlock,
+  tokenMonitored,
+  channelOpened,
+  channelDeposited,
+  channelClosed,
+} from 'raiden-ts/channels/actions';
+import { matrixPresenceUpdate } from 'raiden-ts/transport/actions';
+import { raidenReducer } from 'raiden-ts/reducer';
+import { pathFindServiceEpic } from 'raiden-ts/path/epics';
+import { pathFound, pathFind, pathFindFailed } from 'raiden-ts/path/actions';
+
+import { epicFixtures } from '../fixtures';
+import { raidenEpicDeps } from '../mocks';
+
+describe('PFS: pathFindServiceEpic', () => {
+  const depsMock = raidenEpicDeps();
+  const {
+    token,
+    tokenNetwork,
+    channelId,
+    partner,
+    settleTimeout,
+    isFirstParticipant,
+    txHash,
+    state,
+    matrixServer,
+    partnerUserId,
+  } = epicFixtures(depsMock);
+
+  const target = '0x0100000000000000000000000000000000000005' as Address,
```

Comment:
Might make sense to move this to `epicFixtures()` so we don't accumulate a lot of new test data here over time